lib/task_helpers/imports/roles.rb

```diff
@@ -0,0 +1,31 @@
+module TaskHelpers
+  class Imports
+    class Roles
+      def import(options = {})
+        return unless options[:source]
+
+        glob = File.file?(options[:source]) ? options[:source] : "#{options[:source]}/*.yaml"
+        Dir.glob(glob) do |fname|
+          begin
+            roles = YAML.load_file(fname)
+            import_roles(roles)
+          rescue => e
+            $stderr.puts "Error importing #{fname} : #{e.message}"
+          end
+        end
+      end
+
+      private
+
+      def import_roles(roles)
+        roles.each do |r|
+          r['miq_product_feature_ids'] = MiqProductFeature.all.collect do |f|
```

Comment:
It would be better to pull the `MiqProductFeature.all` out of the loop and put it in a var so that it doesn't get called for every iteration.